lib/vm/src/instance.rs

```diff
@@ -186,7 +186,7 @@ mod test {
 
         let init_used = orig_gas - instance.get_gas();
         println!("init used: {}", init_used);
-        assert_eq!(init_used, 70533);
+        assert_eq!(init_used, 44675);
```

Comment:
What is causing this reduction in init? Do we charge for storage already? Or is it the simplified `State` serialization in

```rust
    deps.storage.set(
        CONFIG_KEY,
        &to_vec(&State {
            verifier: deps.api.canonical_address(&msg.verifier)?,
            beneficiary: deps.api.canonical_address(&msg.beneficiary)?,
            funder: params.message.signer,
        })
        .context(SerializeErr { kind: "State" })?,
    );
    Ok(Response::default())
```

Reply:
This is due to not creating the `[1, 34, 217, 78, ...` arrays for canonical address. Both in state as well as in Params/Env. Crazy expensive, huh? You see why I try to remove raw bytes (not just for aesthetics)